testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/display/MiniAppDisplayActivity.kt

```diff
@@ -129,14 +132,9 @@ class MiniAppDisplayActivity : BaseActivity() {
                 AppSettings.instance.urlParameters
```

Comment:
Do I also need to launch first-time screen when displaying miniapp by url?

Reply:
Hmm, I think probably not. The mini app from a URL won't have a manifest, so we won't be able to display the first time permissions.